script/Arcueid/Day 10/10_00_ARC10_7_2B.txt

```diff
@@ -21,7 +21,7 @@ If that's so―――if I search for a place without lines while looking over th
 [sha:d097092e5c1e16d6205645da4b92cc330c34c54f]{
 -- Page 775, Offset 26390.
 -- 短絡的だが、これが今の俺に出来る唯一の手段だ。
-It's far too simplistic, but this is the only means I have as of now.
+This feels like a rudimentary strategy, but it's all of got.
```

Comment:
I've got

Reply:
these typos are always the most embarrassing. Like how did my wires get so crossed that I thought "of" was "I've"